### PR TITLE
List of starred documents

### PR DESCRIPTION
**Pull request by nschloe**
starred

To be merged once the corresponding backend branch has landed.

### CODE REVIEW

File: app/components/starred-docs.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                  '/people/' + id + '/stars'
+              );
+            } catch (err) {
+              notificationService.notifications.push({
+                type: 'error',
+                message: err.data.message ? err.data.message :
+                  'could not fetch starred documents (unknown reason)'
+              });
+            }
+            ctrl.starredDocuments = ret.data.documents;
```
Comment (andrenarchy) [inline app/components/starred-docs.ts]: shouldn't this be inside the `try` block? Then `ret` could also move there and be declared as `const`...

Comment (nschloe) [inline app/components/starred-docs.ts]: i guess it doesn't matter

Comment (andrenarchy) [inline app/components/starred-docs.ts]: `ret` is undefined in the case of a failing http request. Then accessing `ret.data.documents` will throw an error.

**Comment by nschloe** [comment]
@andrenarchy Error handling improved.

**Pull request merged**